addons/addons.rst

```diff
@@ -214,3 +214,25 @@ Symptoms: Some langpack submissions fail, but some succeed. Reruns work after so
 Workaround: Rerun each failed task after waiting some time. Contact AMO about rate limiting for our user.
 
 Current fix: Our current addonscript AMO user is supposed to bypass rate limiting checks. This should work unless something changes or breaks.
+
+Intermittent errors
+~~~~~~~~~~~~~~~~~~~
+
+2020.08.24: a `langpack task <https://firefox-ci-tc.services.mozilla.com/tasks/J_VRZ2YWRU2Iyfwarovc3A/runs/0>`_ ran out of retries for ``bg`` and ``az``. A rerun fixed it::
```

Comment:
I'm not really sure what value this section is adding, tbh. It feels like it should be an Issue filed on addonscript. At least, I wouldn't think to look at the docs for something like this (maybe that says more about our docs than this patch though).

r=me either way, but that's my thoughts.

Reply:
Yeah, not sure either, other than "I'm hitting this error, what do I do?" -> oh, rerun til success.